app/src/main/java/com/example/calculator/Calculator.kt

```diff
@@ -0,0 +1,46 @@
+package com.example.calculator
+
+class Calculator {
```

Comment:
у класса нет состояния, можно сделать 
```suggestion
object Calculator {
```